.devcontainer/openmower_config.bash

```
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+override_env_file=/opt/ws/.devcontainer/override/.env
+
+# Values below are defaults. You can override them by creating a file in .devcontainer/override/.env
+# File will be sourced here.
+
+if [ -f "$override_env_file" ]; then
+    echo "Sourcing override environment file: $override_env_file"
+    source "$override_env_file"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)